app/utils/isMobileViewport.ts

```diff
@@ -1,8 +1,24 @@
 import { useEffect, useState } from 'react';
 
 const isMobileViewport = () => {
-  const width = window.innerWidth;
-  return width < 768;
+  if (!__CLIENT__) {
+    return false;
+  }
+  const width = window?.innerWidth;
+
+  // Check user agent string
+  const userAgent = navigator.userAgent.toLowerCase();
+  const isMobileUA =
+    /mobile|android|iphone|ipad|ipod|blackberry|iemobile|opera mini/i.test(
+      userAgent,
+    );
```

Comment:
Can we also check the User-Agent header when SSR?

Reply:
seems to work fine? 🤷🏼‍♂️